rack-mount/catalog/120mm-fan-tray_mod.scad

```diff
@@ -0,0 +1,61 @@
+use <../tray/tray.scad>
+
+/*
+  Simple tray for a beelink mini-s series mini pc.

```

Comment:
The comments here should probably be changed. Same for the other files too.

Reply:
Updated comments to reflect the mini PC or device the tray is designed for, will upload soon.